contracts/base/BaseEscrowDapp.sol

```diff
@@ -2,16 +2,15 @@ pragma solidity 0.5.17;
 
 // Contracts
 import "@openzeppelin/contracts-ethereum-package/contracts/ownership/Ownable.sol";
+import "@openzeppelin/contracts-ethereum-package/contracts/token/ERC20/SafeERC20.sol";
 import "./BaseUpgradeable.sol";
 
-// Interfaces
-import "@openzeppelin/contracts-ethereum-package/contracts/token/ERC20/IERC20.sol";
-
 // Libraries
 import "../util/AddressArrayLib.sol";
 
 contract BaseEscrowDapp is Ownable, BaseUpgradeable {
     using AddressArrayLib for address[];
+    using SafeERC20 for IERC20;
```

Comment:
You never use any of the SafeERC20 functions in this contract. I dont think its needed :)

Reply:
Ah yes, the Uniswap and Compound apps inherit from this base escrow, which is why this was here. I've imported SafeERC20 into each of them and removed this.